### ISSUE TITLE
Add output options

### ISSUE THREAD

**Issue by davisshaver** [OpenNewsLabs/field-guide-open-source-newsroom#53]
This is such a nice piece of content that folks might want to read it on a Kindle, iPad, or another device.

I tried to find a PDF link (Read the Docs _should_ be making one of these automatically) but no dice.

**Comment by ryanpitts** [comment]
Yes! We'll absolutely be enabling PDF/EPUB downloads for this. Our plan is to wait for another round of community contributions and suggestions, so we can fill in some final gaps and add some more case studies. Once people download a static version, seems pretty likely they won't come back often to check for new versions, so we'd like to make sure the first downloadable edition is a little more complete.
